specs/toolbar.md

```diff
@@ -0,0 +1,127 @@
+<fast-toolbar
+    orientation="horizontal" 
+    aria-labelledby="label"
+>
+    <div>
+        <fast-button>Bold</fast-button>
+        <fast-button>Italic</fast-button>
+        <fast-button>Underline</fast-button>
+    </div>
+    <select>
+        <option>Option 1</option>
+        <option>Option 2</option>
+        <option>Option 2</option>
+    </select>
+    <div>
+        <fast-button>More</fast-button>
+    </div>
+</fast-toolbar>
+```
+
+**With invisible label:**
+```html
+<fast-toolbar
+    orientation="horizontal" 
+    aria-label="Toolbar label"
+>
+    <div>
+        <fast-button>Bold</fast-button>
+        <fast-button>Italic</fast-button>
+        <fast-button>Underline</fast-button>
+    </div>
+    <select>
+        <option>Option 1</option>
+        <option>Option 2</option>
+        <option>Option 2</option>
+    </select>
+    <div>
+        <fast-button>More</fast-button>
+    </div>
+</fast-toolbar>
+```
+
+### Accessibility
+
+The interaction model for the toolbar should map to the [W3C interaction model](https://w3c.github.io/aria-practices/#toolbar).
+
+#### Secondary navigation methods
+
+The W3C specification gives the following affordance to toolbar keyboard interactions:
+
+Horizontal:
+
+> In horizontal toolbars, Left Arrow and Right Arrow navigate among controls. Up Arrow and Down Arrow can duplicate Left Arrow and Right Arrow, respectively, or can be reserved for operating controls, such as spin buttons that require vertical arrow keys to operate.
+
+Vertical:
+
+> In vertical toolbars, Up Arrow and Down Arrow navigate among controls. Left Arrow and Right Arrow can duplicate Up Arrow and Down Arrow, respectively, or can be reserved for operating controls, such as horizontal sliders that require horizontal arrow keys to operate.
+
+While affordances are given for secondary navigation methods, the initial stance of FAST-DNA is to not support the methods mentioned above in order to not complicate/convolute the API or cause conflict with interaction models for controls contained within the toolbar (such as a select or dropdown). As we gain feedback on the control, we can look at how we may want to support the above if/when it is requested.
+
+#### Disabled controls
+
+> Typically, disabled elements are not focusable when navigating with a keyboard. However, in circumstances where discoverability of a function is crucial, it may be helpful if disabled controls are focusable so screen reader users are more likely to be aware of their presence.
+
+Following the "flexible" principle of FAST-DNA, I think we should support an API which enables exposing disabled controls to users. This allows implementors to decide if their circumstances warrant exposing that functionality or not. A simple solution here is that if an author wants to expose a disabled control, they would simply add it to the tabindex:
+
+```html
+<fast-button disabled tabindex="0">Disabled</fast-button>
+```
+
+#### Labels
```

Comment:
Would it make sense to surface a "label" slot for this?

```
<fast-toolbar>
   <h3 slot="label">This is a toolbar</h3>
   ....
</fast-toolbar>
```

Reply:
We certainly could as a convenience - I don't know that it gives a huge amount of value here compared to other places, but it doesn't hurt.

Reply:
yea just consistency I think